content/en/user-guide/tools/docker-desktop-extension/index.md

```diff
@@ -1,43 +1,58 @@
 ---
-title: "Docker Desktop Extension"
+title: "Docker Extension"
```

Comment:
nit: perhaps slightly rename?
```suggestion
title: "LocalStack Docker Extension"
```

Also, do we want the filename/URL path continue to be `docker-desktop-extension`?

Really not critical, also happy to merge as-is - just wondering which name we want to use, as the URL will be sticky (and added as a link to the Extension UI, etc)